docs/links.rst

```diff
@@ -21,20 +21,20 @@
     Note: this isn't working for me with multiword terms with spaces.
 
 
-.. _Pyramid manual: http://docs.pylonsproject.org/projects/pyramid/en/1.3-branch/index.html
+.. _Pyramid manual: http://docs.pylonsproject.org/projects/pyramid/en/latest/narr/project.html
```

Comment:
Applied to wrong line. See below.

This should be:
http://docs.pylonsproject.org/projects/pyramid/en/latest/